dioptas/widgets/plot_widgets/HistogramLUTItem.py

```diff
@@ -281,6 +300,18 @@ def setLevels(self, mn, mx):
     def empty_function(self, *args):
         pass
 
+    def _configurationButtonClicked(self):
+        dialog = ColormapDialog()
+        dialog.setCurrentGradient(self.gradient.saveState())
+        dialog.setRange(*self.getExpLevels())
+        dialog.sigCurrentGradientChanged.connect(self._configurationGradientChanged)
+        dialog.sigRangeChanged.connect(self.setLevels)
+        dialog.exec()
```

Comment:
The dialog is modal to spare updating the dialog when the main window is updated, but that could be added.